VirtoCommerce.Storefront.Model/StaticContent/JsonPage.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Newtonsoft.Json.Linq;
+
+namespace VirtoCommerce.Storefront.Model.StaticContent
+{
+    public class JsonPage
```

Comment:
I suggest to do not use the new `JsonPage` type for designer pages, need to use already exists static content types such as `ContentPage` and `BlogArticle` instead.
The `ContentPage` type has properties `Template` and `MeatInfo` that can have the same meaning as `Layout` and `Settings` in `JsonPage`

Reply:
remove it too